src/tests/ftest/datamover/copy_basics.py

```diff
@@ -0,0 +1,455 @@
+
+        # pool1 -> pool2
+        new_uuid = self.gen_uuid()
+        self.run_dcp(
+            src="/", dst="/",
+            src_pool=pool1, src_cont=container1,
+            dst_pool=pool2,
+            override_dst_cont=new_uuid,
+            test_desc="copy_auto_create_dest (2)")
+        self.setup_and_run_ior("DFS", self.flags_read, "/", pool2, new_uuid)
+
+    def setup_and_run_ior(self, api, flags, path, pool, cont_uuid):
+        self.ior_cmd.api.update(api)
+        self.ior_cmd.flags.update(flags)
+        test_file = os.path.join(path, self.test_file)
+        self.ior_cmd.test_file.update(test_file)
+        if pool:
+            self.ior_cmd.set_daos_params(self.server_group, pool, cont_uuid)
+        else:
+            self.ior_cmd.dfs_pool.update(None)
+            self.ior_cmd.dfs_cont.update(None)
+            self.ior_cmd.dfs_group.update(None)
+            self.ior_cmd.dfs_svcl.update(None)
+        out = self.run_ior(self.get_ior_job_manager_command(), self.processes)
+
+    def write_location(self, loc):
+        """ Writes the test data using ior """
+        ior_params = loc.get_ior_params()
+        self.setup_and_run_ior(
+            ior_params.api,
+            self.flags_write,
+            ior_params.path,
+            ior_params.pool,
+            ior_params.cont_uuid)
+
+    def read_verify_location(self, loc):
+        """ Read verifies the test data using ior """
+        ior_params = loc.get_ior_params()
+        self.setup_and_run_ior(
+            ior_params.api,
+            self.flags_read,
+            ior_params.path,
+            ior_params.pool,
+            ior_params.cont_uuid)
+
+    def run_dcp(self, src, dst,
+                prefix=None,
+                src_pool=None, dst_pool=None, src_cont=None, dst_cont=None,
+                src_svc=None, dst_svc=None,
+                override_dst_cont=None,
+                test_desc=None):
+        """Use mpirun to execute the dcp utility"""
+        # param for dcp processes
+        processes = self.params.get("processes", "/run/datamover/*")
+
+        # Set up the dcp command
+        dcp = DataMover(self.hostlist_clients)
+        dcp.get_params(self)
+        dcp.daos_prefix.update(prefix)
+        dcp.src_path.update(src)
+        dcp.dest_path.update(dst)
+        dcp.set_datamover_params(src_pool, dst_pool, src_cont, dst_cont)
+
+        # Set the service levels. Useful for UNS paths, 
+        # since set_datamover_params doesn't set this without a pool.
+        if src_svc is not None:
+            dcp.daos_src_svcl.update(src_svc)
+        if dst_svc is not None:
+            dcp.daos_dst_svcl.update(dst_svc)
+
+        # Handle manual overrides
+        if override_dst_cont is not None:
+            dcp.daos_dst_cont.update(override_dst_cont)
+
+        # Run the dcp command
+        if test_desc is not None:
+            self.log.info("Running dcp: {}".format(test_desc))
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments